ts/Core/Axis/Axis.ts

```diff
@@ -5485,6 +5485,18 @@ class Axis implements AxisComposition, AxisLike {
             }
         }
 
+        // If min is bigger than highest,
+        // or if max less than lowest value,
+        // the chart should not render points. (#14417)
+        if (axis.min !== null && axis.max !== null) {
+            if (axis.min > axis.max) {
```

Comment:
These two conditions can be combined into one.

Reply:
Thank you Torstein, will do the changes and prepare the PR after resolving the failing tests that are occuring.